src/pages/Home/Home.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import {ThisDay} from "./components/ThisDay/ThisDay";
+import {ThisDayInfo} from "./components/ThisDayInfo/ThisDayInfo";
+import {Days} from "./components/Days/Days";
+import {HomeWrapper} from "./HomeStyle";
+
+interface Props {
+
+}
+ export const Home = (props: Props) => {
```

Comment:
Call components with Component in end of name(Container if it's container)
Props aren't
```suggestion
 export const HomeComponent = () => {
```